content/en/docs/collector/internal-telemetry.md

```diff
@@ -113,3 +118,148 @@ journalctl | grep otelcol | grep Error
+| `otelcol_receiver_accepted_spans`                 | Number of spans successfully ingested and pushed into the pipeline.                       | Counter   |            |
+| `otelcol_receiver_refused_log_records`            | Number of logs that could not be pushed into the pipeline.                                | Counter   |            |
+| `otelcol_receiver_refused_metric_points`          | Number of metric points that could not be pushed into the pipeline.                       | Counter   |            |
+| `otelcol_receiver_refused_spans`                  | Number of spans that could not be pushed into the pipeline.                               | Counter   |            |
+| `otelcol_rpc_client_duration`                     | Measures the duration of outbound RPC.                                                    | Histogram | `detailed` |
+| `otelcol_rpc_client_request_size`                 | Measures the size of RPC request messages (uncompressed).                                 | Histogram | `detailed` |
+| `otelcol_rpc_client_requests_per_rpc`             | Measures the number of messages received per RPC. Should be 1 for all non-streaming RPCs. | Histogram | `detailed` |
+| `otelcol_rpc_client_response_size`                | Measures the size of RPC response messages (uncompressed).                                | Histogram | `detailed` |
+| `otelcol_rpc_client_responses_per_rpc`            | Measures the number of messages sent per RPC. Should be 1 for all non-streaming RPCs.     | Histogram | `detailed` |
+| `otelcol_rpc_server_duration`                     | Measures the duration of inbound RPC.                                                     | Histogram | `detailed` |
+| `otelcol_rpc_server_request_size`                 | Measures the size of RPC request messages (uncompressed).                                 | Histogram | `detailed` |
+| `otelcol_rpc_server_requests_per_rpc`             | Measures the number of messages received per RPC. Should be 1 for all non-streaming RPCs. | Histogram | `detailed` |
+| `otelcol_rpc_server_response_size`                | Measures the size of RPC response messages (uncompressed).                                | Histogram | `detailed` |
+| `otelcol_rpc_server_responses_per_rpc`            | Measures the number of messages sent per RPC. Should be 1 for all non-streaming RPCs.     | Histogram | `detailed` |
+| `otelcol_scraper_errored_metric_points`           | Number of metric points the Collector failed to scrape.                                   | Counter   |            |
+| `otelcol_scraper_scraped_metric_points`           | Number of metric points scraped by the Collector.                                         | Counter   |            |
+
+### Events observable with internal logs
+
+The Collector logs the following internal events:
```

Comment:
I tested on-the-fly reconfiguration with my own Collector instance, and it did not work, so I removed "The Collector reconfigured" from this list. @open-telemetry/collector-approvers, could you let me know if there are any other events in this list for which logging has not yet been implemented? Thanks!